generate_figures/deep_sets_linear_profiling_var_n.py

```diff
@@ -1,74 +1,33 @@
 import wandb
 import pandas as pd
 import matplotlib.pyplot as plt
-import wandb
-from matplotlib import cm
-import yaml
-import os
-from mpl_toolkits.axes_grid1.inset_locator import zoomed_inset_axes, mark_inset
 
-fontsize = 10
-ticksize = 14
-figsize = (8, 3.5)
-params = {
-    "text.usetex": True,
-    "font.family": "serif",
-    "figure.figsize": figsize,
-    "figure.dpi": 80,
-    "figure.edgecolor": "k",
-    "font.size": fontsize,
-    "axes.labelsize": fontsize,
-    "axes.titlesize": fontsize,
-    "xtick.labelsize": ticksize,
-    "ytick.labelsize": ticksize,
-}
+from mpl_toolkits.axes_grid1.inset_locator import zoomed_inset_axes, mark_inset
+from utilities import get_plot_params, get_results_by_tag
 
 output_dir = "./figures"
 plot_name = "deep-sets-linear-profiling-var-n"
-
 output_path = output_dir + "/" + plot_name + ".pdf"
 
 api = wandb.Api()
+project = "highdimensionaleconlab/symmetry_dynamic_programming"
 
-sym_runs = "highdimensionaleconlab/symmetry_dynamic_programming"
-def satisfying_runs(tag):
-    run_num=0 
-    d=[]
-    overall_tag = api.runs(sym_runs, filters={"tags": tag})
-    
-    for i in range(len(overall_tag)):
-        run_num+=1
-        try: 
-            get = dict(overall_tag[i].summary)
-            get['seed'] = int(overall_tag[i].config.get('seed'))
-            get['N'] = int(overall_tag[i].config.get('N'))
-        except:
-            print("THERE IS NO RUN DATA ASSOCIATED")
-        else:
-            d.append(get) 
-    return(pd.DataFrame.from_records(d))
-
-df_deep = satisfying_runs("baseline_deep_sets_N")            
-
-##once we have retcode will be something like 
-#df_successes= df_deep.loc[df_deep['ret_code']==0]
-
-
+params = get_plot_params((8, 3.5), 10, 14)
```

Comment:
Typically it is better to move this sort of thing closer to the "call site" where they are used, and/or to call the variable something more specific.  For example `plot_params` so we don't think it is something like a deep model parameters datasttructure.